### PR TITLE
Dtspb 2772 zipservice

### PR DESCRIPTION
**Pull request by srochani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/probate/service/evidencemanagement/builder/DocumentManagementURIBuilder.java [hunk 1]
```diff
@@ -26,4 +26,8 @@ public String buildUrl() {
     public String buildUrl(String id) {
         return buildUrl() + "/" + id;
     }
+
+    public String buildBinaryUrl(String id) {
```
Comment (srochani) [inline src/main/java/uk/gov/hmcts/probate/service/evidencemanagement/builder/DocumentManagementURIBuilder.java]: Refactor buildUrl

File: src/main/java/uk/gov/hmcts/probate/service/evidencemanagement/header/IdamHttpHeaderFactory.java [hunk 2]
```diff
@@ -43,4 +45,13 @@ public HttpHeaders getAuthorizationHeaders() {
         headers.setContentType(MediaType.APPLICATION_JSON);
         return headers;
     }
+
+    @Override
+    public HttpHeaders getAuthorizationHeadersForDocuments() {
```
Comment (srochani) [inline src/main/java/uk/gov/hmcts/probate/service/evidencemanagement/header/IdamHttpHeaderFactory.java]: Rename to download

File: src/main/java/uk/gov/hmcts/probate/service/evidencemanagement/upload/EmUploadService.java [hunk 3]
```diff
@@ -69,4 +74,15 @@ public void expire(Document document) throws JsonProcessingException {
             new HttpEntity<>(json, headers.getApplicationJsonHttpHeader()),
             HashMap.class);
     }
-}
\ No newline at end of file
+
+    public ByteArrayResource getDocument(String id) {
```
Comment (srochani) [inline src/main/java/uk/gov/hmcts/probate/service/evidencemanagement/upload/EmUploadService.java]: Rename to getDocumentByteArray

File: src/main/resources/application.yml [hunk 4]
```diff
@@ -517,6 +517,9 @@ welsh:
 iht-estate:
   switch-date: ${EXCEPTED_ESTATE_DATE_OF_DEATH:2022-01-01}
 
+storage:
+  connection: DefaultEndpointsProtocol=https;AccountName=fileshareprobate;AccountKey=NUxpuQd9G/vCFi2bMk3roxhmUg2lXBmkXG8LKpY6HbEhAjMxUjQC4HLCQneIoRU0FySpZLCz6xuM+AStLcvnow==;EndpointSuffix=core.windows.net
```
Comment (srochani) [inline src/main/resources/application.yml]: Transfer into vault

File: src/test/resources/application.yml [hunk 5]
```diff
@@ -415,6 +415,9 @@ welsh:
 iht-estate:
   switch-date: "2022-01-01"
 
+storage:
+  connection: DefaultEndpointsProtocol=https;AccountName=fileshareprobate;AccountKey=NUxpuQd9G/vCFi2bMk3roxhmUg2lXBmkXG8LKpY6HbEhAjMxUjQC4HLCQneIoRU0FySpZLCz6xuM+AStLcvnow==;EndpointSuffix=core.windows.net
```
Comment (srochani) [inline src/test/resources/application.yml]: Transfer into vault

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Comment by srochani** [comment]
Closing and opening another PR

**Pull request closed without merging**